### ISSUE TITLE
Deprecated selector in `atom-delek-syntax/index.less`

### ISSUE THREAD

**Issue by rubin110** [ambv/atom-delek-syntax#2]
I like this theme, however I'm running into a small issue with 1.15.0 beta 3...

> Deprecated calls
> No deprecated calls
> Deprecated selectors
> atom-delek-syntax
> CHECK FOR UPDATEDISABLE PACKAGE
> atom-delek-syntax/index.less
> Starting from Atom v1.13.0, the contents of atom-text-editor elements are no longer encapsulated within a shadow DOM boundary. This means you should stop using :host and ::shadow pseudo-selectors, and prepend all your syntax selectors with syntax--. To prevent breakage with existing style sheets, Atom will automatically upgrade the following selectors:
> 
> Etc, etc.

**Comment by ambv** [comment]
You're right. Sadly, I don't have free cycles these days to address it. I'd definitely merge a pull request to that effect.
